conda-environment.yml

```diff
@@ -39,5 +38,5 @@ dependencies:
   - GeoAlchemy2 >= 0.17.0
   - xarray >=0.9
   - toolz
-  - odc-geo
+  - odc-stac >= 0.4.0
```

Comment:
Shouldn't these versions/packages more or less be a carbon copy of the dependencies in pyproject.toml?